fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb

```diff
@@ -36,8 +38,10 @@ def run!
         OptionParser::MissingArgument => e
         abort e.to_s
       rescue FastlaneCore::Interface::FastlaneError => e # user_error!
+        collector.did_raise_error(@program[:name])
         display_user_error!(e, e.message)
       rescue => e # high chance this is actually FastlaneCore::Interface::FastlaneCrash, but can be anything else
+        collector.did_raise_error(@program[:name])
```

Comment:
Yeah correct, we're passing a string here. To make our API more robust, we could also just call `name.to_sym` inside the method: https://github.com/fastlane/fastlane/pull/4473/files#diff-861844ac6c8a585e117d3135bd82679cR15